invenio_pidstore/cli.py

```diff
@@ -37,6 +40,28 @@
     from flask_cli import with_appcontext
 
 
+def loader_import(ctx, param, value):
+    """Import the loader."""
+    try:
+        return import_string(value)
+    except ImportError:
+        raise click.BadParameter('Loader {0} not found.'.format(value))
+
+
+def process_minter(ctx, param, value):
+    """Return the minter."""
+    with ctx.obj.load_app().app_context():
```

Comment:
@hachreak it loads the app ones but it creates the `app_context` ...

The `__call__` is using minter and it's called multiple times.